samples/model-builder/noxfile.py

```diff
@@ -180,6 +180,7 @@ def blacken(session: nox.sessions.Session) -> None:
 # format = isort + black
 #
 
+
```

Comment:
My linter made some formatting changes. Can remove if required.

Reply:
PR lint is complaining :S